Backpack/src/main/java/net/skyscanner/backpack/calendar/view/MonthView.kt

```diff
@@ -99,7 +99,11 @@ internal class MonthView @JvmOverloads constructor(
   private val monthHeaderSize: Int = ResourcesUtil.dpToPx(52, context)
 
   private var rowHeight: Int = context.resources.getDimensionPixelSize(R.dimen.bpkSpacingXl) * 2
-  private var labelsYOffset: Int = context.resources.getDimensionPixelSize(R.dimen.bpkSpacingXl)
+  private var rowHeightLabeled: Int = context.resources.getDimensionPixelSize(R.dimen.bpkSpacingXxl) +
```

Comment:
That's the value which provides 4dp gap to the bottom line, which is required by design.

Reply:
As we require a very specific spacing it might be more readable to just hard code it?